isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -913,6 +929,15 @@ namespace Isis {
     }
   }
 
+  /**
+   * This method will write an OriginalLabel object.
+   * to the cube as specified by the contents of the Blob object.
+   *
+   * @param Original label data to be written
+   */
+  void Cube::write(OriginalLabel lab) {
+    write(*(lab.toBlob()));
+  }
```

Comment:
I know this write method makes things easier but I don't know how necessary it is and that's something I have been going back and forth on. It'd be good for us to figure out exactly how we want to be writing these blobs back out. While the `*(lab.toBlob())` is a little messy I don't think it's bad to have `write(*(lab.toBlob()))` all over the place rather than `write(lab)`